internal/datastore/crdb/crdb.go

```diff
@@ -74,23 +74,30 @@ func newCRDBDatastore(url string, options ...Option) (datastore.Datastore, error
 		return nil, fmt.Errorf(errUnableToInstantiate, err)
 	}
 
-	poolConfig, err := pgxpool.ParseConfig(url)
+	readPoolConfig, err := pgxpool.ParseConfig(url)
 	if err != nil {
 		return nil, fmt.Errorf(errUnableToInstantiate, err)
 	}
+	writePoolConfig := readPoolConfig.Copy()
```

Comment:
when I was testing locally, it seemed like this didn't create a copy with all of the values. I had to `pgxpool.ParseConfig(url)` again to get a working config.